### ISSUE TITLE
CellML support: the CellML API wrongly allows code generation for non-runnable models

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#2240]
@nickerso reported this issue: if you open [this model](https://github.com/opencor/opencor/files/3899365/bob.txt) and switch to the `Simulation Experiment` view, you will get something like:

<img width="1867" alt="Screen Shot 2019-11-28 at 2 24 52 PM" src="https://user-images.githubusercontent.com/602265/69769718-e1691080-11ea-11ea-9aff-e84dc8191cb8.png">

In other words, no variable of integration is listed in the list of parameters. Yet, we can run the model without any problem. Instead, the `Simulation Experiment` view should tell us that the model doesn't have a variable of integration (since it has a private interface of `in`).

**Comment by agarny** [comment]
After some investigation, the problem is related to the CellML API (!!). The CeVAS service was modified to [allow for incomplete models](https://tracker.physiomeproject.org/show_bug.cgi?id=3176) (see changes to the CellML API codebase [here](https://github.com/opencor/cellml-api/commit/99bd47b8d010c57a4eb78f7ff466f3526599bc72)).

Now, when it comes to OpenCOR, we don't want that "new" behaviour, but the "old" one. The "old" behaviour is actually the one used by OpenCell 0.8, resulting in the aforementioned model being (rightly) considered as non-runnable.

Bottom line: we should revert [the changes](https://github.com/opencor/cellml-api/commit/99bd47b8d010c57a4eb78f7ff466f3526599bc72) introduced to the CellML API, and use that "fixed" version in OpenCOR.
